portfolio/src/main/java/com/google/sps/validators/TextValidator.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.validators;
+
+public interface TextValidator {
+  // Allow only alphanumeric chars and few extra symbols.
+  public static final String validPattern = "^([A-z]|[0-9]|[\\?!,.(\" ])*$";
+
+  public Boolean validate();
```

Comment:
Why not use a primitive `boolean` here? This isn't perfomance-critical but `boolean` is usually the right choice in Java and there's autoboxing which will take care of cases where Boolean is needed.